src/components/GlobalFilter/tagsApi.ts

```diff
@@ -147,7 +213,7 @@ export async function getAllWorkloads({ activeTags, registeredWith }: TagFilterO
       undefined,
       undefined,
       undefined,
-      1, // number of items per page
```

Comment:
These comments are quite useful due to the structure of the arguments. Can you keep them?